dapper/tools/rounding.py

```diff
@@ -93,6 +96,10 @@ def __str__(self):
         if n < 0:
             # Ensure we get 1.30 ±0.01, NOT 1.3 ±0.01:
             frmt = "%%0.%df" % -n
+
+        if n >= 0 and not np.isnan(c) and not np.isinf(c):
```

Comment:
```suggestion
        elif np.isfinite(c):
```
(the nan check is already taken care of on line 82, but we do need to check for infinity)